src/2023/styles/styles.css

```diff
@@ -143,12 +143,14 @@ h2,
 h3,
 h4,
 h5,
-p,
-ol,
-ul {
+p {
   margin: 0;
 }
 
+.ul-padding {
+  padding-left: 15px;
```

Comment:
Could we use `list-style-position: inside;`?

https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position